tests/tensor/nnet/test_sigm.py

```diff
@@ -110,6 +110,7 @@ def setup_method(self):
 
     def test_elemwise(self):
         utt.verify_grad(softplus, [np.random.rand(3, 4)])
+        utt.verify_grad(softplus, [np.linspace(-40, 40, 20)])
```

Comment:
In this repository, we're moving away from performing wasteful, time consuming, shotgun-like numerical tests, so we need to combine these `verify_grad`s into a single one that explicitly tests only important and/or necessary values.